src/styles/main.scss

```diff
@@ -1,3 +1,68 @@
 body {
   margin: 0;
 }
+
+$start-days:
+  mon 0,
+  tue 1,
+  wed 2,
+  thu 3,
+  fri 4,
+  sut 5,
+  sun 6;
+$cell-size: 100px;
+$space-size: 1px;
+$full-width: 720px;
+$cell-color: #eee;
+$choosed-color: pink;
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  padding: 20px 10px;
+  width: $full-width;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $space-size;
+
+  &__day {
+    width: $cell-size;
+    height: $cell-size;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    box-sizing: border-box;
+    background-color: $cell-color;
+    border: 1px solid #000;
+    transition: background-color 500ms, transform 500ms;
+
+    &:hover {
+      cursor: pointer;
+      background-color: $choosed-color;
+      transform: translateY(-20px);
+    }
+
+    @for $day from 1 through 30 {
```

Comment:
I believe the month may have 31 days

Make sure to test your calendar for each possible month's length